### PR TITLE
Feature/sass

### LINKED ISSUE #8: SASS Files
- [ ] Go through existing .scss files and update to SASS syntax
- [ ] create index.scss file
- [ ] create variables.scss file
- [ ] create mixins.scss file

### PR DESCRIPTION
**Pull request by hayleyw7**
# Thank you for pushing your code!
## What (if any) features are you implementing?
- Closes #8 
- Change vanilla CSC to SASS

## What (if anything) did you refactor?
- Some of the CSS in general

## Were there any issues that arose?
- N/A

## Is there anything that you need from your teammate?
- Does the layout still look the same?

## Any other comments, questions, or concerns?
- I'm sorry about all of the commits! I was making so many new changes that I wanted to keep having safe saves on GH and not just locally. They are small commits, and I'm glad to explain to the instructors.

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?
- [x] open index.html
- [ ] console.log()
- [x] dev tools

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by darlaevans2000 — APPROVED** [review]
Looks great! You did an amazing job converting to sass.

File: src/css/base.scss [hunk 1]
```diff
@@ -1,6 +1,56 @@
-/* This is an example of using Sass, otherwise plain CSS works too*/
-$primary-background: pink;
+// COLORS
 
-body {
-  background-color: $primary-background;
```
Comment (williamphelps13) [inline src/css/base.scss:5]: Snagged  9 variables and 5 mixins! The comments definitely help with the organization!

File: src/css/base.scss [hunk 2]
```diff
@@ -1,6 +1,56 @@
...
+$dark-gray: #4d4e4f;
```
Comment (williamphelps13) [inline src/css/base.scss:3]: If we have time to circle back would we want to call the variables more like the general location of the colors rather then the color itself - in the Sass lesson it was "header" or background" as the variable. May not make sense for this project though.

**Review by MTCawthray — APPROVED** [review]
This makes it easy to adjust some of the stylings in the base.scss file. Great job!

File: src/css/styles.scss [hunk 3]
```diff
@@ -1,250 +1,191 @@
...
-  text-align: center;
-  background-color: #DFE166;
-  font-size: 1em;
-  margin: .5em;
-  font-family: sans-serif;
-  display: flex;
-  flex-direction: column;
-  justify-content: flex-start;
-  width: 50%;
+  @include card-history();
```
Comment (williamphelps13) [inline src/css/styles.scss:157]: Amazing how much code can be deleted with this mixin

**Review by williamphelps13 — APPROVED** [review]
Great job! Just one note on variable names to possibly revisit later

**Comment by williamphelps13** [comment]
Also this closes #8

**Pull request merged**